src/plugin.js

```diff
@@ -138,6 +138,18 @@ class PlaylistMenuItem extends Component {
     const li = document.createElement('li');
     const item = this.options_.item;
 
+    if (item.data) {
+      const dataEntries = Object.entries(item.data);
+
+      if (dataEntries.length > 0) {
+        dataEntries.forEach(entry => {
+          const [key, value] = entry;
+
+          li.setAttribute(`data-${key}`, value);
```

Comment:
It may be better to use `dataset` here: https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes#JavaScript_access